docs/manual/common/guide/logging/code/log4j2-lagom-default.xml

```diff
@@ -12,7 +12,7 @@
 
     <Console name="STDOUT">
       <PatternLayout>
-        <Pattern>%d{yyyy-MM-dd'T'HH:mm:ss.SSS'Z'}{GMT+0} %highlight{%level} %logger{0} [%mdc] - %msg%n</Pattern>
+        <Pattern>%d{yyyy-MM-dd'T'HH:mm:ss.SSS'Z'}{GMT+0} %highlight{%level} %logger{10} [%mdc] - %msg%n</Pattern>
```

Comment:
I mentioned on the commit message. While reading the docs I realised that there is a textual reference to that. 

At some point there was a decision to only display 10 lines in console. I believe that this was a thoughtful decision (contrary to mine to set it to 0). So, I though it was better not to change.